tests/smoke/0000_fillseq.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+source smoke/common.sh
+
+DB_BENCH_PARAMS="--benchmarks=fillseq --num=$NUM --value_size=$VALUE_SIZE --histogram $FS_PARAMS $DB_BENCH_EXTRA_PARAMS"
```

Comment:
We may also add params to do direct IO, right ?
rocksdb_use_direct_io_for_flush_and_compaction=ON
rocksdb_use_direct_reads=ON